.github/workflows/usage_guide.yml

```diff
@@ -0,0 +1,34 @@
+name: Publish Usage Guide
```

Comment:
Ideally we keep the two workflows in sync with s2n-quic: https://github.com/aws/s2n-quic/blob/main/.github/workflows/book.yml

* The book should be built on PR's, just not published
* If the PR is opened in the main repo, it would be nice to push the book to S3 so you can preview it before merging
* We should be using the same publish action - not sure which one is better

Reply:
Good suggestion! Not sure how to keep the workflows in sync though, if they're in two different repos.